### ISSUE TITLE
Windows dev setup

### ISSUE THREAD

**Issue by callmeashish** [doccano/doccano#566]
How to reproduce the problem
Installing using docker compose on Windows.

$ git clone https://github.com/doccano/doccano.git
$ cd doccano
$ docker-compose -f docker-compose.dev.yml up

----------------------------------------

Trying to setup dev environment using docker compose.
Error in powershell:

ERROR: for doccano_backend_1  Cannot start service backend: OCI runtime create failed: container_linux.go:346: starting container process caused "process_linux.go:449: container init caused \"rootfs_linux.go:58: mounting \\\"/var/lib/docker/volumes/doccano_venv/_data\\\" to rootfs \\\"/var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged\\\" at \\\"/var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged/src/venv\\\" caused \\\"mkdir /var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged/src/venv: file exists\\\"\"": unknown

ERROR: for backend  Cannot start service backend: OCI runtime create failed: container_linux.go:346: starting container process caused "process_linux.go:449: container init caused \"rootfs_linux.go:58: mounting \\\"/var/lib/docker/volumes/doccano_venv/_data\\\" to rootfs \\\"/var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged\\\" at \\\"/var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged/src/venv\\\" caused \\\"mkdir /var/lib/docker/overlay2/60285993abd46b610ecfd9dbe612b213e898e9f993d1b6e7056e568db1458ef8/merged/src/venv: file exists\\\"\"": unknown
ERROR: Encountered errors while bringing up the project.

Your Environment
---------
*   Operating System: Windows
*   Python Version Used: 3.6
*   When you install doccano: 16th Feb 2020

**Comment by icoxfog417** [comment]
Could you please confirm that you can mount the volume? I tried Windows10/Python 3.6/Docker v19.03.5, it works fine.

**Comment by callmeashish** [comment]
Hey @icoxfog417 ,
Thanks for the quick response.
![image](https://user-images.githubusercontent.com/15687230/74606660-4e7c9b80-50a0-11ea-9c75-b57da1d73632.png)

I have mounted my C drive as volume and there is no other drive.
Should I be mounting the exact folder that I cloned my github repo into?
Is there any prerequisite packages that I should have besides docker ?

Thanks again, appreciate the help.
